src/docs/funding.md

```diff
@@ -0,0 +1,51 @@
+## Funding
+
+The Simple Standard for Sharing Ontological Mappings (SSSOM) is a community-driven project which has received support from many different sources.
+We list the most important ones in the following.
+
+### Volunteering efforts
+
+A huge fraction of the work on SSSOM has been done by volunteers without dedicated grant support.
+We hereby acknowledge their contributions as being absolutely essential. A selection of amazing contributions (by no means exhaustive):
+
+- The development of [SSSOM Java](https://incenp.org/dvlpt/sssom-java/)
```

Comment:
I am not sure this is necessary (and I would never have added that myself), but I’d be lying if I said that I do not appreciate it. Thanks!

Reply:
It stays!